tasks/rpmbuild-test

```diff
@@ -30,25 +31,26 @@ uuidgen > file.lock
 cp file.lock uuid.saved
 while true; do
      # Check if lock exists on remote server
-     while [[ $(rsync --dry-run -avz file.lock fedora-atomic@artifacts.ci.centos.org::fedora-atomic/${RSYNC_BRANCH}/repo/lockdir) != *"file.lock"* ]]; do
+     while [[ $(rsync --ignore-existing --dry-run -avz file.lock fedora-atomic@artifacts.ci.centos.org::fedora-atomic/${RSYNC_BRANCH}/repo/lockdir) != *"file.lock"* ]]; do
           sleep 60
      done
+     cp uuid.saved file.lock
```

Comment:
this is pretty hacky...  I think I would combine the above rsync with an -e "file.lock" test.